app/Models/Role.php

```diff
@@ -125,4 +128,9 @@ public static function getRootAdmin(): self
 
         return $role;
     }
+
+    public function nodes(): BelongsToMany
+    {
+        return $this->belongsToMany(Node::class, RoleNode::class, 'role_id', 'node_id', 'id', 'id');
```

Comment:
Since we're using the expected column names, can we shorten this?

```suggestion
        return $this->belongsToMany(Node::class, RoleNode::class);
```